### DISCUSSION TITLE
PDF2Markdown 效果评估

### DISCUSSION THREAD

**Discussion by DayDreamerEric** [magicpdf/Magic-PDF#895] [Q&A] [answered]
首先，感谢开源～～

想问一下：
1. PDF2Markdown之后，有无开源数据集可以用来最终生成的markdown的效果？
2. 假设已有PDF文件对应的Golden Markdown，具体的评估应该如何进行呢？

**Comment by myhloli** [comment, marked as answer]
https://github.com/opendatalab/OmniDocBench

**Reply by DayDreamerEric** [reply]
Get & Thanks～
